src/components/Sum.jsx

```diff
@@ -1 +1,11 @@
-// export function Sum() {}
+import React from 'react';
+
+export const Sum = ({ a = 0, b = 0 }) => {
```

Comment:
You need to set default values ​​in defaultProps

Reply:
I didn't find it in the checklist, so destructuring also can be used here